en/identity-server/next/docs/apis/organization-apis/restapis/org-application-mgt.yaml

```
@@ -292,6 +284,71 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/ApplicationPatchModel'
+            example:
+              name: "pickup"
+              description: "This is the configuration for Pickup application."
+              applicationVersion: "v1.0.0"
+              imageUrl: "https://example.com/logo/my-logo.png"
+              clientId: "fd52e37f-1b8b-44f5-ad1a-5d846ad03b18"
+              isManagementApp: false
+              isB2BSelfServiceApp: false
+              applicationEnabled: true
+              associatedRoles:
+                allowedAudience: "ORGANIZATION"
```

```suggestion
                allowedAudience: "APPLICATION"
```

Since the recommendation is application, and I hope no roles need to be added in next part to funtion this.

We don't expect to have `roles` to be passed with app creation or update after IS-7

---

This is now updated.